webapp/Connector/src/view/Header.js

```diff
@@ -119,3 +119,86 @@ Ext.define("Connector.view.Header", {
         this.callParent();
     }
 });
+
+Ext.define('Connector.view.AnnouncementHeader', {
+
+    extend : 'Ext.panel.Panel',
+
+    layout : {
+        type : 'hbox',
+        align : 'middle'
+    },
+
+    alias : 'widget.announcementheader',
+
+    hidden : true,
+
+    cls: 'announcementheader',
+
+    initComponent : function() {
+
+        this.msg = this.getMessages();
+        this.items = [
+            {
+                xtype: 'box',
+                tpl: ['<img src="{imgSrc}" width="20" height="20">'],
+                data: {imgSrc: Connector.resourceContext.imgPath + '/' + 'warning_indicator.svg'},
+                margin : 15
+            },
+            this.msg,
+            {
+                xtype : 'button',
+                text : 'x',
+                margin : 15,
+                tooltip : 'dismiss this notification',
+                handler : function(){
+                    LABKEY.Ajax.request({
+                        url : LABKEY.ActionURL.buildURL("core", "dismissCoreWarnings.api"),
+                        method : 'POST',
+                        scope : this,
+                        success : function (){
+                            this.setVisible(false);
+                        },
+                        failure : function(){
+                            Ext.Msg.alert('Failed to dismiss warnings');
+                        }
+                    });
+                },
+                scope : this
+            }
+        ];
+
+        LABKEY.Ajax.request({
+            url: LABKEY.ActionURL.buildURL("cds", "getDismissableWarnings.api"),
+            method: 'GET',
+            scope : this,
+            success: function (response){
+                var o = Ext.decode(response.responseText);
+                if (o.messages) {
+                    this.setVisible(true);
+                    this.msg.update(o.messages);
+                }
+                else
+                    this.setVisible(false);
```

Comment:
I assume this is not needed since the component has a default state of `hidden: true`.